src/modules/Lan.rb

```diff
@@ -643,10 +643,20 @@ def Import(settings)
       end
 
       NetworkConfig.Import(settings["config"] || {})
+
       # Ensure that the /etc/hosts has been read to no blank out it in case of
       # not defined <host> section (bsc#1058396)
       Host.Read
 
+      # FIXME: hotfix for Host::ResolveHostnameToStaticIPs - the functionality is
```

Comment:
In general hhe relation between the Host and connections hostname has to be reevaluated, at least the method description suggested that the method for using the hostname associated to the static IPs was installation specific. 

At the end it is also expected in AutoYaST which makes sense when there is no `<hosts>` section